task/06-conditions-n-loops-tasks.js

```diff
@@ -150,7 +177,9 @@ function doRectanglesOverlap(rect1, rect2) {
  *   
  */
 function isInsideCircle(circle, point) {
-    throw new Error('Not implemented');
+    if ((point.x - circle.center.x) * (point.x - circle.center.x) + 
+    (point.y - circle.center.y) * (point.y - circle.center.y) < circle.radius * circle.radius) return true; 
+    return false; 
```

Comment:
Try to use Math.hypot or Math.pow methods to solve this task 
He have the operator ** as well